packages/components/organisms/f-loyalty/src/components/HowItWorks.vue

```diff
@@ -0,0 +1,585 @@
+                            <template v-for="({ value }, i) in bags">
+                                <p
+                                    :key="`${i}_percent`"
+                                    :class="$style['c-howItWorks-example-percentages-value']">
+                                    {{ value }}
+                                </p>
+                                <p
+                                    v-if="i < (bags.length - 1)"
+                                    :key="`${i}_percent_icon_plus`"
+                                    :class="$style['c-howItWorks-example-percentages-icon']">
+                                    +
+                                </p>
+                                <p
+                                    v-else
+                                    :key="`${i}_percent_icon_equals`"
+                                    :class="$style['c-howItWorks-example-percentages-icon']">
+                                    =
+                                </p>
+                            </template>
+                        </div>
+                    </div>
+                </div>
+                <div :class="$style['c-howItWorks-example-total']">
+                    <p :class="$style['c-howItWorks-example-total-text']">
+                        {{ $t('howItWorks.exampleSection.total') }}
+                    </p>
+                    <h3 :class="$style['c-howItWorks-example-total-price']">
+                        {{ total }}
+                    </h3>
+                </div>
+            </div>
+        </f-card>
+
+        <h3 :class="$style['c-howItWorks-mediaTitle']">
+            {{ $t('howItWorks.media.title') }}
+        </h3>
+
+        <div :class="$style['c-howItWorks-mediaWrapper']">
+            <template v-for="({ title, text, image }, i) in mediaCards">
+                <f-card
+                    :key="i"
+                    has-outline>
+                    <media-element
+                        role="presentation"
+                        :title="title"
+                        :text="text"
+                        :flex="flexLayout"
+                        content-align="center"
+                        image-align="center"
+                        text-size="md"
+                        :image-url="image"
+                        :alt-text="title" />
+                </f-card>
+            </template>
+        </div>
+
+        <div :class="$style['c-stampcards-terms']">
+            <a
+                :class="$style['c-stampcards-terms-link']"
+                data-test-id="terms-and-conditions"
+                :href="$t('termsUrl')">
+                {{ $t('termsText') }}
+            </a>
+        </div>
+    </div>
+</template>
+
+<script>
+import FCard from '@justeat/f-card';
+import MediaElement from '@justeat/f-media-element';
+
+const TEMP_IMG_PATH = 'https://d30v2pzvrfyzpo.cloudfront.net/b/hw/img/decoration/';
```

Comment:
Note this will only work on average half the time due to relying on the pool that homeweb is active in - let's try and get cloudinary online soon as we can.